sdk/containerinstance/Azure.ResourceManager.ContainerInstance/api/Azure.ResourceManager.ContainerInstance.netstandard2.0.cs

```diff
@@ -432,6 +439,7 @@ public partial class ContainerInstanceUsage
     {
         internal ContainerInstanceUsage() { }
         public int? CurrentValue { get { throw null; } }
+        public string Id { get { throw null; } }
```

Comment:
What's the format of this property? Is it a GUID or resource identifier?
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#d4cebe383870410cab2e14d0709f78bb).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#9f6fc0ae7c8e497e98d7979dadf97a4c).

Reply:
From this API, https://learn.microsoft.com/en-us/rest/api/container-instances/location/list-usage?tabs=HTTP#code-try-0, it looks like it is a resource identifier. However, this was auto generated and we have not made changes to this API. I guess the SDK change for this property was not generated before and the latest swagger picked it up. I'm not comfortable changing it since I'm not the owner of the API, hence I do not know what would break. Would like to keep changes restricted to Priority property and confidential containers. @fnuarnav  - Can you please help tag Mainak here?

Reply:
This seems like some sort of "pseudo resource" identifier. While it "looks" a little bit like an ARM resource id in terms of the structure of the string, there isn't any actual backing resource here.

For example, this is what the `id` looks like in a sample response:

```json
            "id": "/subscriptions/ec9fcd04-e188-48b9-abfc-a35d515f1836/providers/Microsoft.ContainerInstance/locations/eastus/usages/ContainerGroups",
```

But this resource doesn't really exist:

```
GET https://management.azure.com/subscriptions/ec9fcd04-e188-48b9-abfc-a35d515f1836/providers/Microsoft.ContainerInstance/locations/eastus/usages/ContainerGroups?api-version=2022-10-01-preview

-> 404

{
    "message": "No HTTP resource was found that matches the request URI 'https://provider-eastus.console.azure.com/providers/api/subscriptions/ec9fcd04-e188-48b9-abfc-a35d515f1836/providers/Microsoft.ContainerInstance/locations/eastus/usages/ContainerGroups?api-version=2022-10-01-preview'."
}
```

So `string` seems to be the best approximation of this API, the way it currently works (it isn't really a resource identifier or a guid). 

@archerzz -- I propose we move forward with this as is. Hopefully this addresses your concerns.

Reply:
@diyellap @mlalic Thanks!